prober/http.go

```diff
@@ -49,7 +49,10 @@ func probeHTTP(ctx context.Context, url string, want []byte) error {
 		return fmt.Errorf("fetching %q: status code %d, want 200", url, resp.StatusCode)
 	}
 
-	bs, err := io.ReadAll(&io.LimitedReader{resp.Body, maxHTTPBody})
+	bs, err := io.ReadAll(&io.LimitedReader{
```

Comment:
I would just use the constructor: https://pkg.go.dev/io#LimitReader

and keep it on one line like you had it

Reply:
Oops, duh, fixed.